README.md

```diff
@@ -243,7 +287,7 @@ Thus both targets and services are implemented according to systemd
 using units in standard locations such as /usr/lib/systemd/system and
 /etc/systemd/system.
```

Comment:
For paths, commands, and constants, I like to emphasize them by quoting them in backticks. GitHub renders this with a fixed-width font and light gray background.